### DISCUSSION TITLE
Ship Lat / Lon / Alt / Deg circle not populating

### DISCUSSION THREAD

**Discussion by dualznz** [iaincollins/icarus#16] [Bugs & Problems]
Version: 0.13.5

Im not sure if this has been mentioned but in the ship tab the circle that contains Lat / Lon / Alt / Deg is not populating and is greyed out, is this still in dev ..

Also nice app by the way.

**Comment by iaincollins** [comment]
Hi there,

Yes ICARUS Terminal is currently in pre-release, and as such builds have many known issues (and as a general rule, not looking for bug reports).

That said, I don't know of any bugs with that feature. It may not be clear from the UI but that information is only available - both in ICARUS Terminal and in game - when on (or approaching) a planet surface.

I'll think about if this is something the UI can convey more clearly.
